src/main/webapp/app/exam/manage/student-exams/student-exams.component.html

```diff
@@ -5,15 +5,22 @@ <h2>
                 <span jhiTranslate="artemisApp.studentExams.title">Student exams</span>
             </h2>
         </div>
-        <div class="ml-auto d-flex">
-            <button class="btn btn-primary" style="height: 40px;" (click)="generateStudentExams()" *ngIf="course?.isAtLeastInstructor" [disabled]="isLoading">
-                <span *ngIf="isLoading" class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
-                <span jhiTranslate="artemisApp.studentExams.generateStudentExams">Generate student exams</span>
-            </button>
-            <button class="btn btn-primary ml-1" style="height: 40px;" (click)="startExercises()" *ngIf="course?.isAtLeastInstructor" [disabled]="isLoading">
-                <span *ngIf="isLoading" class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
-                <span jhiTranslate="artemisApp.studentExams.startExercises">Start Exercises</span>
-            </button>
+        <div class="ml-auto">
+            <div class="d-flex mb-1">
+                <button class="btn btn-primary ml-1" style="height: 40px;" (click)="handleGenerateStudentExams()" *ngIf="course?.isAtLeastInstructor" [disabled]="isLoading">
+                    <span *ngIf="isLoading" class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+                    <span jhiTranslate="artemisApp.studentExams.generateStudentExams">Generate student exams</span>
+                </button>
+                <button class="btn btn-primary ml-1" style="height: 40px;" (click)="generateMissingStudentExams()" *ngIf="course?.isAtLeastInstructor" [disabled]="isLoading">
```

Comment:
I believe it would be useful if the button `Generate missing individual exams` is only clickable/visible if there are students without exams.
As you already track that, this should be quite simple to do, right?

Reply:
Button now deactivates if all students have an exam